scripts/prepare_eez_data/prepare_eez_data.py

```diff
@@ -0,0 +1,108 @@
+import geopandas as gpd
+import json
+import argparse
+import logging
+import pandas as pd
+
+# Configure logging
+logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def load_csv(csv_file):
+    try:
+        csv_data = pd.read_csv(csv_file)
+        return csv_data
+    except FileNotFoundError:
+        logger.error(f"CSV file '{csv_file}' not found.")
+        return None
+
+
+def find_country_and_status_for_sovereign(row, csv_data):
+    country_status_mapping = {}
+
+    for i in range(1, 3):  # Assuming there are up to 5 SOVEREIGN columns (SOVEREIGN1 to SOVEREIGN5)
+        sovereign_col = f'SOVEREIGN{i}'
+        sovereign_value = row[sovereign_col]
+
+        if pd.notna(sovereign_value):
+            match = csv_data[csv_data['Country'] == sovereign_value]
+            if not match.empty:
+                status_col = f'SOVEREIGN{i}_ABS_STATUS'
+                status_value = match['Status'].values[0]
+
+                # Convert int64 to Python integer
+                status_value = int(status_value)
+
+                country_status_mapping[status_col] = status_value
+
+    return country_status_mapping
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Shapefile to JSON Converter")
+    parser.add_argument("shapefile_path", type=str, nargs="?", help="Path to the input shapefile")
+    parser.add_argument("output_json_file", type=str, nargs="?", help="Path to the output JSON file")
```

Comment:
Could be nice to send to standard out if a file isn't specified here, like:
```suggestion
    parser.add_argument("output_json_file", type=str, nargs="?", help="Path to the output JSON file", default=sys.stdout)
```
That way you can easily pipe it to tools like `jq` or `grep` for debugging / checking.